### PR TITLE
adding readme and updating add inventory route

### PR DESCRIPTION
**Pull request by joesho112358**

### CODE REVIEW

File: Inventory.py [hunk 1]
```diff
@@ -1,24 +1,27 @@
-from flask import Flask, render_template, url_for, request
+from flask import Flask, render_template, url_for, request, redirect
 import sqlite3
 
-#connection to DB
-conn = sqlite3.connect('Inventory.db')
-
-#setup cursor object
-cursor = conn.cursor()
```
Comment (joesho112358) [inline Inventory.py:8]: doing this was throwing an error for me about accessing sqlite from the wrong thread, moved the connection into the methods

File: Inventory.py [hunk 2]
```diff
@@ -34,19 +37,18 @@ def main():
...
         length = request.form['length']
         height = request.form['height']
         width = request.form['width']
-        invcount = request.form['invCount']
-        add_item(itemnumber, proddescrip, weight, length, height, width, invcount)
-        return render_template('add_inventory.html',)
-
+        inv_count = request.form['invCount']
+        add_item(item_number, product_description, weight, length, height, width, inv_count)
+        return redirect('/')
```
Comment (joesho112358) [inline Inventory.py:51]: redirecting to home page to see created data

File: templates/add_inventory.html [hunk 3]
```diff
@@ -14,7 +14,7 @@ <h1> Please fill in the selected information below and click "Submit" to create
 <div class="grid-container">
         <div class="grid-child" style="border:solid 2px; padding: 20px;">
             <h1 style="text-align: center">Please fill in the following specifications</h1>
-            <form action="/" method="POST">
+            <form action="add_inventory" method="POST">
```
Comment (joesho112358) [inline templates/add_inventory.html:17]: the form was pointed to the wrong method in the controller

File: Inventory.py (hunk 1 above)
Comment (cbjfan2009) [reply Inventory.py:8]: I did receive a database locked issue; perhaps this was the culprit and I misunderstood the traceback?

**Pull request merged**